### PR TITLE
e2e test for simplify report closure

### PR DESCRIPTION
**Pull request by vishalpandeyvip**
[Screencast from 25-01-23 07:36:03 PM IST.webm](https://user-images.githubusercontent.com/60269359/214586045-79accb39-0715-4380-8c16-dfee28fe5f4a.webm)

### CODE REVIEW

**Review by DhaaraniCIT — APPROVED** [review]

File: cypress/e2e/002-journeys/001-onboarding.cy.js [hunk 1]
```diff
@@ -79,6 +79,8 @@ describe('onboarding journey', () => {
   }
 
   it('completes onboarding QBO for a workspace', () => {
+    cy.visit('/workspaces/onboarding/employee_settings')
```
Comment (ashwin1111) [inline cypress/e2e/002-journeys/001-onboarding.cy.js:82]: let's not hard code it, cy.visit('/') should automatically take you to this url

File: cypress/e2e/002-journeys/001-onboarding.cy.js [hunk 2]
```diff
@@ -21,21 +20,22 @@ describe('onboarding journey', () => {
...
     cy.selectConfigurationField(2, 'Checking')
     cy.selectConfigurationField(3, 'Report')
     cy.selectConfigurationField(4, 'Current Date')
 
     // Expand non-reimbursable expenses section
     cy.enableConfigurationToggle(1)
-    cy.selectConfigurationField(5, 'Credit Card Purchase')
-    cy.selectConfigurationField(6, 'Visa')
+    cy.selectConfigurationField(5, 'Payment Processing')
+    cy.selectConfigurationField(6, 'Bill')
```
Comment (ashwin1111) [inline cypress/e2e/002-journeys/001-onboarding.cy.js:34]: let's retain the same selection options as before. Lot of tests depend on these settings.

**Review by DhaaraniCIT — APPROVED** [review]

File: cypress/coverage/coverage-summary.json [hunk 3]
```diff
@@ -1,113 +0,0 @@
-{"total": {"lines":{"total":7404,"covered":6713,"skipped":0,"pct":90.66},"statements":{"total":7470,"covered":6771,"skipped":0,"pct":90.64},"functions":{"total":1514,"covered":1347,"skipped":0,"pct":88.96},"branches":{"total":3576,"covered":2927,"skipped":0,"pct":81.85},"branchesTrue":{"total":0,"covered":0,"skipped":0,"pct":100}}
```
Comment (ashwin1111) [inline cypress/coverage/coverage-summary.json:1]: run `npm run cypress_run`, that will run e2e tests once for all files and update coverage from this file

Comment (ashwin1111) [reply cypress/coverage/coverage-summary.json:1]: on local

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**